### PR TITLE
added whiteboard

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/queueWithStacks/queue-with-stacks.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  enqueue(value) {
+    let currentNode = new Node(value);
+    currentNode = this.front;
+    if (this.rear) { this.rear.next = currentNode; }
+    this.rear = currentNode;
+  }
+
+}
+
```
Comment (adrianhuebner) [inline javascript/code-challenges/queueWithStacks/queue-with-stacks.js:22]: You look like you were off to a great start! If you have any questions drop them in the comments here or on Canvas!
